### PR TITLE
Fix logs - make the visable in our app

### PR DESCRIPTION
**Pull request by razbensimon**
show them in the `stderr`

### CODE REVIEW

**Review by ghost — APPROVED** [review]
LGTM

File: libexec/retry [hunk 1]
```diff
@@ -1,5 +1,9 @@
 #!/usr/bin/env bash
 
+function info() {
+  echo -e "\033[0;32m[INFO] ${1}\033[0;39m" >&2
```
[the remark this answers is not in the document]
Comment (ronnyorot) [reply libexec/retry:4]: Like the error

[the remark this answers is not in the document]
Comment (razbensimon) [reply libexec/retry:4]: will be green 
image

[the remark this answers is not in the document]
Comment (ronnyorot) [reply libexec/retry:4]: OK, I validated that it's green? Which is also a weird color for info. shouldn't it be white/gray

**Pull request merged**